### ISSUE TITLE
Dead example link in README

### ISSUE THREAD

**Issue by jrson83** [75lb/command-line-usage#37]
Thanks for making this great tool-set! Is it still actively under maintenance?

 Just noticed, the anchor links [content](https://github.com/75lb/command-line-usage#commandlineusagecontent)  and [optionList](https://github.com/75lb/command-line-usage#commandlineusageoptionlist) links at top of the readme are broken.

**Comment by 75lb** [comment]
Yes, it is actively maintained.. i have fixed the dead links in the readme - let me know if there's anything else..
